src/manifest.json

```diff
@@ -53,11 +51,11 @@
     "__firefox__browser_style": false
   },
 
-  "__chrome|opera__options_page": "options.html",
   "options_ui": {
     "page": "options.html",
     "open_in_tab": true,
-    "__chrome__chrome_style": false
+    "__chrome|opera__chrome_style": false,
+    "__firefox__browser_style": false
```

Comment:
They all have it. Was this intentional? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui

Reply:
no, was not intentional. thanks!